src/mediapipe_calculators/python_executor_calculator.cc

```diff
@@ -88,51 +114,67 @@ class PythonExecutorCalculator : public CalculatorBase {
             RET_CHECK(false);
         }
         nodeResources = it->second;
+        outputsTimestamp = mediapipe::Timestamp(mediapipe::Timestamp::Unset());
         LOG(INFO) << "PythonExecutorCalculator [Node: " << cc->NodeName() << "] Open end";
         return absl::OkStatus();
     }
 
     absl::Status Process(CalculatorContext* cc) final {
         LOG(INFO) << "PythonExecutorCalculator [Node: " << cc->NodeName() << "] Process start";
+        LOG(INFO) << "Input timestamp : " << cc->InputTimestamp();
+        for (const auto& input : cc->Inputs()) {
+            LOG(INFO) << input.Name() << " : " << !input.IsEmpty();
```

Comment:
If we want to keep those logs (we probably do?), we should format it to be more descriptive, for example:
"Received input tag:X name:Y isEmpty:Z for timestamp:W"

Reply:
I think we should remove it. It's on the INFO level, so I'd reduce amount of such logs as much as possible.

Reply:
Mediapipe INFO == OVMS DEBUG just fyi